databases/week3/mealSharing.sql

```diff
@@ -0,0 +1,450 @@
+CREATE Database mealSharing;
+Use mealSharing;
+create Table Meal(
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `title` varchar(255),
+  `description` text,
+  `location` varchar(255),
+  `when` datetime,
+  `max_reservations` int(50),
+  `price` decimal,
+  `created_date` date
+);
+create Table Reservation(
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `number_of_guests` int(50),
+  `meal_id` int(10) unsigned,
+  `created_date` date,
+  `contact_phonenumber` varchar(255),
+  `contact_name` varchar(255),
+  `contact_email` varchar(255),
+  CONSTRAINT `fk_meal_reservation` FOREIGN KEY (`meal_id`) REFERENCES `Meal` (`id`)
+);
+create Table Review(
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `title` varchar(255),
+  `description` text,
+  `meal_id` int(10) unsigned,
+  `stars` int(50),
+  `created_date` date,
+  CONSTRAINT `fk_meal_review` FOREIGN KEY (`meal_id`) REFERENCES `Meal` (`id`)
+);
+INSERT INTO
+  Meal(
+    title,
+    description,
+    location,
+    `when`,
+    max_reservations,
+    price,
+    created_date
+  )
+VALUES
+  (
+    'KFC',
+    'fried chicken',
+    'valby',
+    '2021-07-23',
+    11,
+    200,
+    '2021-02-23'
+  );
```

Comment:
You can also keep the insert statements on 2 lines, it improves readability of the queries.
e.g. 
`INSERT INTO Meal(title, description, location, when, max_reservations, price, created_date )`
`VALUES ('KFC', 'fried chicken', 'valby', '2021-07-23', 11, 200, '2021-02-23');`

Reply:
yeah.. its happenning because of extention...